scripts/tests/twister/test_runner.py

```diff
@@ -1988,11 +1993,13 @@ def test_projectbuilder_report_out(
 
     assert results_mock.cases == 25
 
-    assert all([log in re.sub(
+    trim_actual_log = re.sub(
         r'\x1B(?:[@-Z\\-_]|\[[0-?]*[ -/]*[@-~])',
         '',
         caplog.text
-    ) for log in expected_logs])
+    )
+    trim_actual_log = re.sub(r'twister:runner.py:\d+', '', trim_actual_log)
```

Comment:
```suggestion
    trim_actual_log = re.sub(r'twister:runner.py:\d+', 'twister:runner.py:', trim_actual_log)
```
to remove only line numbers  keeping patterns specific; same in `TESTDATA_13`

Reply:
or maybe better to change `expected_logs` in `TESTDATE_13` to use list with the only text we want to verify (without spaces etc.). Then you do not need to remove line numbers from `trim_actual_log
```suggestion
TESTDATA_13 = [
    (
        'error', True, True, False,
        ['20/25 dummy platform',
         'dummy.testsuite.name',
         'ERROR dummy reason (cmake)'],
        None
    ),
    (
        'failed', False, False, False,
        ['dummy platform',
         'dummy.testsuite.name',
         'FAILED : dummy reason'],
        'INFO    - Total complete:   20/  25  80%  skipped:    3,' \
        ' failed:    3, error:    1'
    ),
    (
        'skipped', True, False, False,
        ['20/25 dummy platform',
         'dummy.testsuite.name',
         'SKIPPED (dummy reason)'],
        None
    ),
    (
        'filtered', False, False, False,
        [],
        'INFO    - Total complete:   20/  25  80%  skipped:    4,' \
        ' failed:    2, error:    1'
    ),
    (
        'passed', True, False, True,
        ['20/25 dummy platform',
         'dummy.testsuite.name',
         'PASSED',
         '(dummy handler type: dummy dut, 60.000s)'],
        None
    ),
    (
        'passed', True, False, False,
        ['20/25 dummy platform',
         'dummy.testsuite.name',
         'PASSED (build)'],
        None
    ),
    (
        'unknown status', False, False, False,
        ['Unknown status = unknown status'],
        'INFO    - Total complete:   20/  25  80%  skipped:    3,' \
        ' failed:    2, error:    1\r'
    ),
    (
        'timeout', True, False, True,
        ['20/25 dummy platform',
         'dummy.testsuite.name',
         'UNKNOWN',
         '(dummy handler type: dummy dut, 60.000s/seed: 123)'],
        None
    ),
]
```

Reply:
@gchwier  thanks for the suggestion, I also think about this way, but this will have a potential on what if ther is some exception chars bedies the target strings to match, I think this case is to ensure the output is thus formated.